### PR TITLE
Documentation changes

### PR DESCRIPTION
**Pull request by gsheni**
Implements changes for compatibility with Featuretools 1.0.0

### CODE REVIEW

**Review by thehomebrewnerd — COMMENTED** [review]
So, I have to ask. The title and description of this MR suggests these are changes for FT >= 1.0 compatibility, but it seems like most of the changes unrelated to that. Did we change direction on this MR, or is there something else going on?

File: .github/workflows/unit_tests_with_latest_deps.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      - main
+
+name: Unit Tests - Latest Dependencies
+jobs:
+  unit_tests:
+    name: Unit Tests - Python ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8"]
```
Comment (thehomebrewnerd) [inline .github/workflows/unit_tests_with_latest_deps.yml:15]: Should be ["3.7", "3.8", "3.9"]

File: autonormalize/examples/example_data_gen.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import csv
-import random
+# import csv
```
Comment (thehomebrewnerd) [inline autonormalize/examples/example_data_gen.py:1]: Do we need this file and the other files in the `examples` directory? Can we delete the directory or do we need to hang on to it for something?

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+* Implement your pull request. If needed, add new tests or update the documentation.
+* Before submitting to GitHub, verify the tests run and the code lints properly
+
+  ```bash
+  # runs test
+  make test
+
+  # runs linting
+
```
Comment (thehomebrewnerd) [inline contributing.md:46]: Add `make lint` command

File: contributing.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  # view docs locally
+  open build/html/index.html
+  ```
+
+#### 3. Submit your Pull Request
+
+* Once your changes are ready to be submitted, make sure to push your changes to GitHub before creating a pull request. Create a pull request, and our continuous integration will run automatically.
+* Update the "Future Release" section of the release notes (`docs/source/release_notes.rst`) to include your pull request and add your github username to the list of contributors.  Add a description of your PR to the subsection that most closely matches your contribution:
+  * Enhancements: new features or additions to Featuretools.
```
Comment (thehomebrewnerd) [inline contributing.md:67]: `Featuretools` -> `AutoNormalize`

File: contributing.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  * Fixes: things like bugfixes or adding more descriptive error messages.
+  * Changes: modifications to an existing part of Featuretools.
```
Comment (thehomebrewnerd) [inline contributing.md:69]: `Featuretools` -> `AutoNormalize`

File: contributing.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  * Documentation Changes
+  * Testing Changes
+
+   Documentation or testing changes rarely warrant an individual release notes entry; the PR number can be added to their respective "Miscellaneous changes" entries.
```
Comment (thehomebrewnerd) [inline contributing.md:73]: Do we still do this? Seems like we almost always include Documentation and Testing changes in the section above. Maybe we should remove. @gsheni Thoughts?

File: contributing.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+* We will review your changes, and you will most likely be asked to make additional changes before it is finally ready to merge. However, once it's reviewed by a maintainer of Featuretools, passes continuous integration, we will merge it, and you will have successfully contributed to Featuretools!
```
Comment (thehomebrewnerd) [inline contributing.md:74]: `Featuretools` -> `AutoNormalize` in two places in this sentence.

File: contributing.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+## Report issues
+
+When reporting issues please include as much detail as possible about your operating system, featuretools version and python version. Whenever possible, please also include a brief, self-contained code example that demonstrates the problem.
```
Comment (thehomebrewnerd) [inline contributing.md:78]: `featuretools` -> `autonormalize`

File: docs/source/guides/demo/liquor/__init__.py [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+
+PWD = join(PWD, 'liquor')
+
+def load_sample():
+    df = read_csv(join(PWD, "Iowa_Liquor_Sales.csv"))
+    # df = df.drop(columns=df.columns[12:])
+    # df = df.drop(range(1500000, 12591077))
+    # df = df.dropna()
+    # df = df.drop_duplicates()
+    # df = df.head(1000)
```
Comment (thehomebrewnerd) [inline docs/source/guides/demo/liquor/__init__.py:13]: Do we need these lines or can we delete?

File: docs/source/install.rst [hunk 10]
```diff
@@ -0,0 +1,115 @@
+Install
+*******
+
+AutoNormalize is available for Python 3.7 and 3.8. The recommended way to install Featuretools is using ``pip`` ::
```
Comment (thehomebrewnerd) [inline docs/source/install.rst:4]: Also Python 3.9?

`Featuretools` -> `AutoNormalize`

File: docs/source/release_notes.rst [hunk 11]
```diff
@@ -0,0 +1,21 @@
+.. _release_notes:
+
+Release Notes
+-------------
+
+.. Future Release
```
Comment (thehomebrewnerd) [inline docs/source/release_notes.rst:6]: Uncomment this section

File: docs/source/release_notes.rst [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+  ==============
+    * Enhancements
+    * Fixes
+    * Changes
+    * Documentation Changes
+      * Updated sphinx documentation and guides
+    * Testing Changes
+      * Updated tests to work with featuretools 1.0
+
+.. Thanks to the following people for contributing to this release:
```
Comment (thehomebrewnerd) [inline docs/source/release_notes.rst:16]: Add yourself (and Gaurav) to the contributors list

**Review by thehomebrewnerd — COMMENTED** [review]
Just a few relatively minor things to clean up.

File: docs/source/guides/demo/liquor/__init__.py (hunk 9 above)
Comment (tuethan1999) [reply docs/source/guides/demo/liquor/__init__.py:14]: they show how to reproduce the sample dataset we provide from the kaggle dataset. I added a comment with that explanation

File: autonormalize/examples/example_data_gen.py (hunk 2 above)
Comment (tuethan1999) [reply autonormalize/examples/example_data_gen.py]: example3 is used in the tests. I deleted the others and just kept the code that generates example3. I could also git rid of the csv and make it a pytest fixture

**Comment by tuethan1999** [comment]
> So, I have to ask. The title and description of this MR suggests these are changes for FT >= 1.0 compatibility, but it seems like most of the changes unrelated to that. Did we change direction on this MR, or is there something else going on?

So a quick search of 'ft. ' shows that the Autonormalize code doesn't rely on any Featuretools apis other than initializing an Entityset which doesn't need to be changed. I think @gsheni started to make a bunch documentation changes in this MR that make this repo more like the other open source repos and I continued that. I can rename the MR to documentation changes instead!

File: docs/source/release_notes.rst [hunk 13]
```diff
@@ -3,19 +3,27 @@
...
 -------------
 
-.. Future Release
+  Future Release
   ==============
     * Enhancements
     * Fixes
     * Changes
     * Documentation Changes
+      * Update release notes and release format
```
Comment (thehomebrewnerd) [inline docs/source/release_notes.rst:12]: I guess we aren't checking for this yet in CI, but would be good to mention the PR number for all the release notes entries we have added, like we do in other libraries.

**Review by thehomebrewnerd — APPROVED** [review]
Just one minor thing, but otherwise seems fine to me!

File: contributing.md (hunk 6 above)
Comment (gsheni) [reply contributing.md:71]: No we don't. We can remove this.

**Review by thehomebrewnerd — APPROVED** [review]

**Pull request merged**